spec/src/main/asciidoc/jsonb.adoc

```diff
@@ -1,3 +1,769 @@
+:sectnums:
+= Jakarta JSON Binding Specification, Version 1.0
+
+Copyright (c) 2019 Eclipse Foundation 
+
 == Introduction
 
-Jakarta JSON Binding defines a binding framework for converting Java(R) objects to and from JSON documents.
+This specification defines binding API between Java objects and https://tools.ietf.org/html/rfc7159[JSON] documents. Readers are assumed to be familiar with JSON; for more information about JSON, see:
+
+* https://www.ics.uci.edu/~fielding/pubs/dissertation/fielding_dissertation.pdf[Architectural Styles and the Design of Network-based Software Architectures]
+* http://rest.blueoxen.net/cgi-bin/wiki.pl[The REST Wiki]
```

Comment:
This link is broken.  It looks like this change was made to get rid of the bibliography and put all of the links inline within the text, which in general is certainly more convenient to the reader.  I think we just need an updated link here.  Otherwise, if the source isn't available anymore, I'm unsure of the correct process for citing it.

Reply:
Good catch, I"ll just remove the citation since it is no longer available